src/script/vco-ie/main_warhammer/wh2_main_skv_clan_moulder/missions.lua

```diff
@@ -118,8 +118,6 @@ local missions = {
 					type DESTROY_FACTION;
 
 					faction wh_dlc08_nor_wintertooth;
-
-					confederation_valid;

```

Comment:
Leave it there. I never tested in on Warhammer III, but in old games it could be buggy if another AI confederated the faction, as you could not "destroy it" anymore. Probably not required anymore, but nice to have just as a safeguard.